salvo-remote/azure-pipelines/README.md

```diff
@@ -0,0 +1,75 @@
+# Salvo Azure Pipelines
+
+Definition of pipelins that run Salvo performance workflows.
+
+## Pipelines
+
+- `salvo_pipelines.yml` defines pipelines for the MVP of Salvo. See the [MVP
+  doc](https://docs.google.com/document/d/15auKcxLfw8iILL7EF4tJ8VrnHce6KiZvd9tzWweT0DY/edit),
+  the [Design
+  doc](https://docs.google.com/document/d/1Qfueli357u4QgOb-7-8RL98N0XnMeu2k6VJDoUwN0A4/edit?resourcekey=0-AyeFMQHHiuajx8JK2w_yfA)
+  and the [Pivot to direct
+  VM](https://docs.google.com/document/d/1auXzV-AEXgMzbtdG06XlZ2d9X5l_XadMheA9h51E7yc/edit).
+
+## Required credentials
+
+### Github PAT
+
+The pipelines use a Github personal access token (PAT) to fetch multiple
+Github repositories.
+
+This token is created based on the [Creating a personal access
+token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token)
+Github guide. It only has read-only access to public repositories.
+
+Once created, the token gets stored on Azure Pipelines as follows:
+1. Go to **Project settings**.
+1. Go to **Service connections**.
+1. Create new or edit the existing service connection **GitHub - mum4k - Salvo MVP**.
+1. Store the token.
+
+Finally the Salvo pipeline on AZP needs access to this service connection. To
+Grant the access:
+1. Go to **Project settings**.
+1. Go to **Service connections**.
+1. Edit the service connection for salvo created by following the steps above.
+1. Click on **Security** under the "three-dot" menu in the top right corner.
+1. List Salvo workflow pipelines under **Pipeline permissions**.
+
+### AzureDevOps PAT
+
+The AzureDevOps PAT is used when downloading binaries built by the Azure
+pipelines. Once downloaded, the binaries are included in the VM disk images
+(AMIs) generated by Packer. This token is created based on the
+[Use personal access
+tokens](https://learn.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate)
+AZP guide.
+
+Once created, the PAT needs to be stored in an AZP **Variable Group** accessible
+by the Salvo pipeline. To edit the variable group on AZP:
+1. Go to **Pipelines** in the left context menu.
+1. Go to **Library**.
+1. Edit variable group **Salvo workflow credentials**.
+1. Store the PAT value in variable **SalvoAzureDevOpsExtPat**.
+
+### AWS access key
```

Comment:
(fyi) This portion will change, we will switch away from the access key to an IAM role to improve security. The current security key that was used to test this no longer exists. I will send a follow-up PR updating it once I develop the IAM approach.